### PR TITLE
Mid September Polaris Sync

### PR DESCRIPTION
**Pull request by Cameron653**
- Uses /TG/ maploader
- Xenomorphs are back
- NEW AND IMPROVED XENOBIO 1 REPLACES XENOBIO 2
- A bunch of other stuff.
- Various bugfixes
- Mice shatter your soul upon death
- Minor text fixes

### CODE REVIEW

**Comment by Ketrai** [comment]
does this mean we get an update? I'd like to coordinate an event with some of the new stuff

**Comment by SpadesNeil** [comment]
>332 files changed

fuck

File: maps/submaps/surface_submaps/forest.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/datum/map_template/surface
+	name = "Surface Content"
```
Comment (SpadesNeil) [inline maps/submaps/surface_submaps/forest.dm]: >HYPE

**Comment by SpadesNeil** [comment]
#YOLO

**Pull request merged**